### DISCUSSION TITLE
There is now a ProjectFNF IRC channel!

### DISCUSSION THREAD

**Discussion by ghost** [aflacc/ProjectFNF#46] [General]
For users of Freenode join #projectfnf
For others go to [here](https://webchat.freenode.net), get a nick (probably should be unique) and put #projectfnf in the channels section. Discord server alternative

**Comment by aflacc** [comment]
po g

**Reply by ghost** [reply]
p og

**Comment by ghost** [comment]
Note: unlike Discord, you have to join every time, and there is no message playback when you rejoin (unless you use a bouncer e.g. bnc4free)
